roles/satosa/templates/db_attribute_store.yaml.j2

```diff
@@ -14,3 +14,5 @@ config:
 
   # Use internal user_id as idp_identifier
   user_id: true
+  blacklist:
+    - '{{ comange_entityid }}'
```

Comment:
Moet hier niet zowel SBS als COmanage worden geblacklist?  Ander krijgt SBS de attirbuten van COmanage binnen, en dat lijkt me niet de bedoeling?  Of begrijp ik niet goed hoe dit werkt?